rev_news/drafts/edition-19.md

```diff
@@ -394,6 +394,7 @@ __Light reading__
 
 
 __Git tools and sites__
+* [Git CI stats](https://larsxschneider.github.io/git-ci-stats/) - List all commits of a given branch and visualize the CI status, CI execution time, and failing test 
 
```

Comment:
Honestly I didn't understand what the link was about before following the link ;-).

If I understand correctly, this is a tool to visualize the travis-CI status for git.git, while the text suggests that it's a tool that anyone can use on his own repo.

My 2 cents,